cmd/rollouts-controller/main.go

```diff
@@ -262,6 +284,7 @@ func newCommand() *cobra.Command {
 	command.Flags().DurationVar(&electOpts.LeaderElectionRenewDeadline, "leader-election-renew-deadline", controller.DefaultLeaderElectionRenewDeadline, "The interval between attempts by the acting master to renew a leadership slot before it stops leading. This must be less than or equal to the lease duration. This is only applicable if leader election is enabled.")
 	command.Flags().DurationVar(&electOpts.LeaderElectionRetryPeriod, "leader-election-retry-period", controller.DefaultLeaderElectionRetryPeriod, "The duration the clients should wait between attempting acquisition and renewal of a leadership. This is only applicable if leader election is enabled.")
 	command.Flags().BoolVar(&selfServiceNotificationEnabled, "self-service-notification-enabled", false, "Allows rollouts controller to pull notification config from the namespace that the rollout resource is in. This is useful for self-service notification.")
+	command.Flags().BoolVar(&onlyAnalysisMode, "only-analysis-mode", false, "Only runs analysis controller")
```

Comment:
Can we change the ergonomics to be more like kubernetes, where specific controllers can be enabled/disabled? e.g. Kubernetes has the following arg to turn on certain controllers:

```
--controllers=namespace,serviceaccount,serviceaccount-token,garbagecollector
```

In our case, we would have:
* rollout
* analysisrun
* experiment

Reply:
Updated cmd flags according to discussion.